FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Plain_Text/DemoTasks/PlaintextMQTTExample.c

```diff
@@ -182,14 +182,16 @@
 /**
  * @brief The length of the outgoing publish records array used by the coreMQTT
  * library to track QoS > 0 packet ACKS for outgoing publishes.
+ * Number of publishes = ulMaxPublishCount * mqttexampleTOPIC_COUNT
```

Comment:
Please add a comment explaining the relation between this count and the publish count "ulmaxpublishcount" with a note that a change in one will lead to a change in the other.